features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/RoomDetailsView.kt

```diff
@@ -80,33 +85,50 @@ fun RoomDetailsView(
             .padding(padding)
             .verticalScroll(rememberScrollState())
         ) {
-            HeaderSection(
-                avatarUrl = state.roomAvatarUrl,
-                roomId = state.roomId,
-                roomName = state.roomName,
-                roomAlias = state.roomAlias
-            )
-
-            ShareSection(onShareRoom = onShareRoom)
+            if (state.dmMember == null) {
```

Comment:
I'd add a computed property in the state instead of checking nullability in the view, like `val showAsDM: Boolean`